sci-physics/feynarts/feynarts-3.11.ebuild

```diff
@@ -0,0 +1,31 @@
+# Copyright 2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+MY_PN=FeynArts
+MY_P=${MY_PN}-${PV}
+
+DESCRIPTION="FeynArts renders Feynman diagrams and generates their topologies."
+HOMEPAGE="http://feynarts.de"
+SRC_URI="http://feynarts.de/${MY_P}.tar.gz"
+
+LICENSE="LGPL-3+"
+SLOT="0"
+KEYWORDS="~amd64"
+
+RDEPEND="
+	sci-mathematics/mathematica
+	"
+DEPEND="${RDEPEND}"
+BDEPEND=""
+
+S="${WORKDIR}/${MY_P}"
```

Comment:
The empty var can be removed. Putting S directly under SRC_URI usually makes things a bit cleaner.